_docs/performance-and-scalability/scalability.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Scalability
+overview: Setup of Istio components to scale horizontally. High availability.
+weight: 60
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+* Setup multiple replicas of the control plane components.
+
+* Setup [Horizontal Pod Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+* Split mixer check and report pods.
+
+* High availability (HA).
+
+* See also [Istio's Performance oriented FAQ](https://github.com/istio/istio/wiki/Istio-Performance-oriented-setup-FAQ)
+
+* And the [Performance and Scalability Working Group](https://github.com/istio/community/blob/master/WORKING-GROUPS.md#performance-and-scalability) work.
+
+Current recommendations (when using all Istio features):
+
+* 1 vCPU per peak thousand requests per second for the sidecar(s)
+
+* Assuming typical cache hit ratio (>80%) for mixer checks: 0.5 vCPU per peak thousand requests per second for the mixer pods.
+
+We plan on providing more granular guidance for customers adopting Istio "A la carte".
+
+The goal for 2018 for Istio is for the overhead/cost of adding it to your application to be less than 15% extra virtual hardware required, we currently (as of 0.7.1) are at around 50% overhead.
```

Comment:
This needs clarification with context. Its best to express the cost as a CPU / latency tax relative to the cost of a non-Istiofied workload.

Saying a "50% overhead" without context is likely to be terrifying to people

Reply:
Yes, the 1 vCPU / k qps above is the cpu tax; the 50% number comes from IBM's blue perf where they estimate it'd take 50% more hardware to run at the same QPS with Istio than without (cc jose/@suryadu )

It's hard to get a precise definition of overhead, as it's very much dependent on your application and setup:
- for instance you could have a single service that process at max cpu for 10s before returning a result; for that setup the Istio overhead would be negligeable
- you could have a in memory data server (kind of like fortio echo server) which does 500k qps on a single machine and if you add istio to it, it would then suddenly need 10-100x more machines

So "10%" overhead is somewhat aspirational for a contrived "average" deployment

Do you want to just scratch the whole paragraph ? or is there a way to keep illustrating our goal ?

(I will add a summary of the latency data which is indirectly shown on
https://deploy-preview-1076--preliminary-istio.netlify.com/docs/performance-and-scalability/synthetic-benchmarks.html )

Reply:
fixed/updated, ptal